package.json

```diff
@@ -34,13 +34,18 @@
     "validate": "npm run typecheck && npm run test && npm run lint && npm run format-check"
   },
   "dependencies": {
+    "@perp/sdk-curie": "^1.6.1-canary.0",
```

Comment:
May want to upgrade to 1.7.0 ? https://github.com/perpetual-protocol/sdk-curie/blob/main/CHANGELOG.md

Reply:
Will have to check if this supports the optimism kovan network we need. The Perp guys said we should use the `1.6.1-canary.0` branch since that was the only one that supported that network. But `1.7.0` looks like an update to that branch so might work. Will give it a shot